portfolio/src/main/java/com/google/sps/servlets/PollDataServlet.java

```diff
@@ -13,28 +13,33 @@
 public class PollDataServlet extends HttpServlet {
 
   private LinkedHashMap<Integer, Integer> bigfootSightings = new LinkedHashMap<>();
```

Comment:
can you add comments to describe what the map contains (e.g. year -> sightings)?